### PR TITLE
Feature/region tiles to pages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard4**
## Summary & Motivation

Moves the data logic from the old design's tiles to the new design's pages; adds appropriate locale strings.

## Detailed design

* Logic for Hospital admissions and positively tested people is fairly straightforward and just copied over.
* Logic for sewer water is made more readable by calculating the data to be used by the charts in separate methods.

## Drawbacks

n/a

## Alternatives

* Logic for the sewer water data could perhaps be moved to a utility file, but it's probably handy to have it close to the consuming charts too.

## Unresolved questions

* I have not fit everything according to the new design files. It is perhaps more logical to do that in a separate PR.

### Governance

- [x] Documentation is added
- [x] Test cases are added or updated
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: src/components/layout/MunicipalityLayout.tsx [hunk 1]
```diff
@@ -48,7 +51,7 @@ export function getMunicipalityLayout() {
 function MunicipalityLayout(props: WithChildren) {
   const { children } = props;
   const router = useRouter();
-  const { data } = useSWR(`/json/NL.json`);
+  const { data } = useSWR(`/json/GM0014.json`);
```
Comment (VWSCoronaDashboard4) [inline src/components/layout/MunicipalityLayout.tsx:54]: Hardcoded, dynamically adjusted stuff is for a different PR.

Comment (VWSCoronaDashboard1) [inline src/components/layout/MunicipalityLayout.tsx:54]:
```suggestion
  const { data } = useSWR<National>(`/json/GM0014.json`);
```
That will statically type the data, saves another cast later on ;)

File: src/pages/gemeente/positief-geteste-mensen.tsx [hunk 2]
```diff
@@ -34,7 +42,68 @@ export function PostivelyTestedPeopleBarScale(props: {
 }
 
 const PostivelyTestedPeople: FCWithLayout = () => {
-  return null;
+  const { data } = useSWR(`/json/GM0014.json`);
```
Comment (VWSCoronaDashboard1) [inline src/pages/gemeente/positief-geteste-mensen.tsx:45]:
```suggestion
  const { data } = useSWR<Municipal>(`/json/GM0014.json`);
```

File: src/pages/gemeente/rioolwater.tsx [hunk 3]
```diff
@@ -1,26 +1,250 @@
...
+  allValues: any[][];
+  averageLabelText: string;
+}
+
+interface SewerWaterBarChartData {
+  keys: string[];
+  data: XrangePointOptionsObject[];
+}
+
+function getSewerWaterMetadata(
```
Comment (VWSCoronaDashboard1) [inline src/pages/gemeente/rioolwater.tsx:52]: There's quite a lot of functions in this component file, can these be put in separate files and imported? That keeps things readable. Also can you add some JSDoc comments to that describe in short what kind processing is performed.

File: src/pages/gemeente/ziekenhuis-opnames.tsx [hunk 4]
```diff
@@ -44,7 +51,58 @@ export function IntakeHospitalBarScale(props: {
 }
 
 const IntakeHospital: FCWithLayout = () => {
-  return null;
+  const { data } = useSWR(`/json/GM0014.json`);
```
Comment (VWSCoronaDashboard1) [inline src/pages/gemeente/ziekenhuis-opnames.tsx:54]:
```suggestion
  const { data } = useSWR<Municipal>(`/json/GM0014.json`);
```

File: src/pages/veiligheidsregio/[code]/positief-geteste-mensen.tsx [hunk 5]
```diff
@@ -35,7 +42,68 @@ export function PostivelyTestedPeopleBarScale(props: {
 }
 
 const PostivelyTestedPeople: FCWithLayout = () => {
-  return null;
+  const { data } = useSWR(`/json/VR01.json`);
```
Comment (VWSCoronaDashboard1) [inline src/pages/veiligheidsregio/[code]/positief-geteste-mensen.tsx:45]:
```suggestion
  const { data } = useSWR<Regionaal>(`/json/VR01.json`);
```

File: src/pages/veiligheidsregio/[code]/rioolwater.tsx [hunk 6]
```diff
@@ -1,15 +1,234 @@
...
+  allValues: any[][];
+  averageLabelText: string;
+}
+
+interface SewerWaterBarChartData {
+  keys: string[];
+  data: XrangePointOptionsObject[];
+}
+
+function getSewerWaterMetadata(
```
Comment (VWSCoronaDashboard1) [inline src/pages/veiligheidsregio/[code]/rioolwater.tsx:52]: Same here. Can these functions live inside their own imports and receive some JSDoc comments? Please? :)

File: src/pages/veiligheidsregio/[code]/ziekenhuis-opnames.tsx [hunk 7]
```diff
@@ -44,7 +51,58 @@ export function IntakeHospitalBarScale(props: {
 }
 
 const IntakeHospital: FCWithLayout = () => {
-  return null;
+  const { data } = useSWR(`/json/VR01.json`);
```
Comment (VWSCoronaDashboard1) [inline src/pages/veiligheidsregio/[code]/ziekenhuis-opnames.tsx:54]:
```suggestion
  const { data } = useSWR<Regionaal>(`/json/VR01.json`);
```

**Review by VWSCoronaDashboard1 — CHANGES_REQUESTED** [review]

File: src/pages/gemeente/positief-geteste-mensen.tsx [hunk 8]
```diff
@@ -1,15 +1,23 @@
...
+import Getest from 'assets/test.svg';
+import formatDecimal from 'utils/formatNumber';
+import { PositiveTestedPeople, Municipal } from 'types/data';
+import useSWR from 'swr';
+import replaceVariablesInText from 'utils/replaceVariablesInText';
 
-const text: typeof siteText.positief_geteste_personen =
-  siteText.positief_geteste_personen;
+const text: typeof siteText.gemeente_positief_geteste_personen =
+  siteText.gemeente_positief_geteste_personen;
```
Comment (VWSCoronaDashboard1) [inline src/pages/gemeente/positief-geteste-mensen.tsx:17]: this typing probably isn't necessary anymore after the changes to the locale typing. But for now I'm ok with it.

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: src/pages/gemeente/positief-geteste-mensen.tsx (hunk 8 above)
Comment (VWSCoronaDashboard4) [reply src/pages/gemeente/positief-geteste-mensen.tsx:17]: we can clean it up in a separate PR perhaps, it's done in many other files as well

File: src/components/layout/MunicipalityLayout.tsx [hunk 9]
```diff
@@ -48,7 +50,7 @@ export function getMunicipalityLayout() {
 function MunicipalityLayout(props: WithChildren) {
   const { children } = props;
   const router = useRouter();
-  const { data } = useSWR(`/json/NL.json`);
+  const { data } = useSWR<Municipal>(`/json/GM0014.json`);
```
Comment (VWSCoronaDashboard) [inline src/components/layout/MunicipalityLayout.tsx:53]: Hardcoded region names? Where does the region ID come from? I'd rather see this work from the get-go.

File: src/components/layout/MunicipalityLayout.tsx [hunk 10]
```diff
@@ -8,6 +8,7 @@ import { getLayout as getSiteLayout } from 'components/layout';
 import { PostivelyTestedPeopleBarScale } from 'pages/gemeente/positief-geteste-mensen';
 import { IntakeHospitalBarScale } from 'pages/gemeente/ziekenhuis-opnames';
 import { SewerWaterBarScale } from 'pages/gemeente/rioolwater';
+import { getSewerWaterBarScaleData } from 'pages/gemeente/_rioolwater.util';
```
Comment (VWSCoronaDashboard) [inline src/components/layout/MunicipalityLayout.tsx:11]: What kind of convention is this? Afaik Next.js does not support or endorse non-pages in the pages folder. The underscore seems to mark it private.

What is the purpose of this file?

File: src/pages/gemeente/_rioolwater.util.ts [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+
+export interface SewerWaterBarScaleData {
+  value: number | undefined;
+  unix: number | undefined;
+  dateInsertedUnix: number | undefined;
+}
+
+export interface SewerWaterLineChartData {
+  averageValues: any[];
+  allValues: any[][];
```
Comment (VWSCoronaDashboard) [inline src/pages/gemeente/_rioolwater.util.ts:31]: Does this need to be more strict?

File: src/pages/gemeente/_rioolwater.util.ts [hunk 12]
```diff
@@ -0,0 +1,241 @@
+import {
```
Comment (VWSCoronaDashboard) [inline src/pages/gemeente/_rioolwater.util.ts:1]: I'm fully convinced this file needs to be moved to the utils folder.

**Review by VWSCoronaDashboard — CHANGES_REQUESTED** [review]
Please extract the utility functions into helper methods in the utils folder.

**Review by VWSCoronaDashboard — APPROVED** [review]

**Pull request merged**